core/src/test/scala/unit/kafka/server/ListOffsetsRequestTest.scala

```diff
@@ -92,19 +93,30 @@ class ListOffsetsRequestTest extends BaseRequestTest {
     assertResponseError(Errors.NOT_LEADER_OR_FOLLOWER, nonReplica, debugReplicaRequest)
   }
 
-  @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
-  def testListOffsetsMaxTimeStampOldestVersion(quorum: String): Unit = {
+  @Test
+  def testListOffsetsMaxTimeStampOldestVersion(): Unit = {
```

Comment:
Why do we remove the original ZK and kraft infra's tests?

Reply:
I add it back, but I think this test is unit test, so we don't need to test it on both zk and kraft modes.

Reply:
I know what you're saying, but let's move the refactor in a separate PR.